reactDashboard/src/App.jsx

```diff
@@ -143,22 +144,22 @@ const App = () => {
   }, [populationMap]);
 
   return (
-    <div className="App">
+    <div className={darkmode ? "App darkmode" : "App"}>
       <Router>
         <ScrollToTop />
         <StopAudio />
 
         <header>
-          <DashboardNavbar />
+          <DashboardNavbar darkmode={darkmode} setDarkmode={setDarkmode} />
         </header>
-
         <Switch>
           <Route exact path={URL_OVERVIEW}>
             <Overview
               councilAreaDataset={councilAreaDataset}
               healthBoardDataset={healthBoardDataset}
               currentTotalsHealthBoardDataset={currentTotalsHealthBoardDataset}
               populationProportionMap={populationProportionMap}
+              darkmode={darkmode}
```

Comment:
At some point we're going to start putting these properties into context rather than passing them down. Not just yet though :)